api/provider/aws/service_list.go

```diff
@@ -9,102 +9,79 @@ import (
 )
 
 func (s *ServiceProvider) List() ([]models.ServiceSummary, error) {
-	clusterArns, err := s.listClusters()
+	clusterNames, err := listClusterNames(s.AWS.ECS, s.Config.Instance())
 	if err != nil {
 		return nil, err
 	}
 
-	clusterServices := map[*string][]*string{}
+	serviceARNs, err := s.listClusterServiceARNs(clusterNames)
+	if err != nil {
+		return nil, err
+	}
 
-	for _, clusterArn := range clusterArns {
-		clusterName := strings.Split(aws.StringValue(clusterArn), ":")[5]
-		clusterName = strings.Replace(clusterName, "cluster/", "", -1)
+	serviceIDs := make([]string, len(serviceARNs))
+	for i, serviceARN := range serviceARNs {
+		// sample service ARN:
+		// arn:aws:ecs:us-west-2:856306994068:task-definition/l0-tlakedev-guestbook-dpl:84
```

Comment:
that's not a service arn

Reply:
Yep, correct. Silly me. :white_check_mark: